pkg/resources/provisioning.cattle.io/v1/cluster/validator_test.go

```diff
@@ -1,12 +1,17 @@
 package cluster
```

Comment:
None of the changes made to the `Admit(...)` function get hit by these tests. This could be handle by updating your current test to call Admit instead of private functions.